### DISCUSSION TITLE
Tags for topics

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [NamelessMC/Nameless#2158] [Ideas]
**Please describe the feature in as much detail as possible**
Besides labels, this can be also cool, to have tags for each topic.

Something like this:
![image](https://user-images.githubusercontent.com/56804505/86518625-0b467d80-be33-11ea-9973-55b33a3e7c76.png)

It can be implemented somewhere around here: *below topic's title - above author's name and posting time*
![image](https://user-images.githubusercontent.com/56804505/86518675-5e203500-be33-11ea-9085-1c9c62fc329d.png)

And also, when you open a topic, it'll show them around here:
![image](https://user-images.githubusercontent.com/56804505/86518700-9758a500-be33-11ea-9145-39eec90c0b23.png)

**Comment by tadhgboyle** [comment]
To be honest, I'm not sure this would fit a forum, but rather a personal blog. Let's see what other developers have to say though. @Derkades @samerton ?

**Comment by Derkades** [comment]
For example, on the spigot forums they use "Solved" to mark help posts that have been solved.

Though where labels are used most (discourse, reddit "flairs") they fully or partly replace forum categories. This forum (discourse) uses a combination of the classic forum/category layout with labels https://forum.storj.io/

**Comment by samerton** [comment]
Would it make more sense to rework labels so you can select multiple, and search by label?

**Comment by MotikaCraft-TheJoshua** [comment]
> Would it make more sense to rework labels so you can select multiple, and search by label?

I firstly wanted to post this as a Feature Request... I thought devs (all of you) not gonna like it.

**Comment by jordi00113** [comment]
> Would it make more sense to rework labels so you can select multiple, and search by label?

I was just looking for an existing feature request for selecting multiple labels. Really like this idea! Would be great to see this feature coming into NamelessMC.

For every category, give the ability to use :
- Only one label out of multiple labels
- Multiple labels without restrictions
- Multiple labels but with a maximum

Don't know what other dev's and users currently think about this

**Edit:**
With the update to pre-8 / pre-9 there is the ability to select multiple labels. But maybe it's an option to implement the things listed above.
